### PR TITLE
Ports: Fix c-ray

### PR DESCRIPTION
**Pull request by linusg**

### CODE REVIEW

File: Userland/Libraries/LibM/math.cpp [hunk 1]
```diff
@@ -445,6 +445,11 @@ double tan(double angle) NOEXCEPT
     return ampsin(angle) / ampsin(M_PI_2 + angle);
 }
 
+float tanf(float angle) NOEXCEPT
+{
+    return (float)tan((double)angle);
```
Comment (awesomekling) [inline Userland/Libraries/LibM/math.cpp:450]: 🤷🏻

Comment (linusg) [reply Userland/Libraries/LibM/math.cpp:450]: what can I say, it has the CxByte stamp of approval though! :smile:

Comment (BenWiederhake) [reply Userland/Libraries/LibM/math.cpp:450]: Are the manual casts really necessary? For me it was sufficient to write `return tan(angle);`

Comment (linusg) [reply Userland/Libraries/LibM/math.cpp:450]: not necessary, but at least this way it's very clear that there's casts both ways :)

**Pull request merged**